src/terms/pairwise.jl

```diff
@@ -56,11 +63,19 @@ end
 
 # This could be factorised with Ewald, but the use of `symbols` would slow down the
 # computationally intensive Ewald sums. So we leave it as it for now.
+# `q` is the phonon `q`-point (`Vec3`), and `ph_disp` a list of `Vec3` displacements to
```

Comment:
This does not really explain much, you need an extended comment here. Explain what this function actually computes when passed a non-nothing q/ph_disp. Write down the formulas? And also that only `forces` is valid in this case?

Reply:
Changed it a bit.